### ISSUE TITLE
AppRegistryNotReady exception with Django 1.7

### ISSUE THREAD

**Issue by apdavison** [open-research/sumatra#199]
Django 1.7 introduced some changes to [standalone scripts](https://docs.djangoproject.com/en/dev/releases/1.7/#standalone-scripts).

Running ```pip install sumatra```, pip installs by default Django 1.7, breaking the current method of "easiest installation". 

To reproduce:

```
virtualenv dj17_test
pip install sumatra
pip install mercurial
mkdir dj17_test
cd dj17_test
hg init
smt init TEST
smtweb
```
gives

```
  File "/users/hoffmann/dev/smt/envs/dj17_test/local/lib/python2.7/site-packages/django/apps/registry.py", line 124, in check_apps_ready
    raise AppRegistryNotReady("Apps aren't loaded yet.")
django.core.exceptions.AppRegistryNotReady: Apps aren't loaded yet.
```

        

 _Imported from Bitbucket issue [213](https://bitbucket.org/apdavison/sumatra/issue/213),  created by Felix11H on 10-22-2014 at 14:42, last modified: 10-22-2014 at 14:46_

**Comment by apdavison** [comment]
In similar fashion ([Issue 191](https://bitbucket.org/apdavison/sumatra/issue/191/stop-using-setup_environ)), version 0.5.2 was incompatible with Django 1.6. Does it make sense to restrict which Django version should be installed through pip (if this is possible?). 
        

_Trac comment by Felix11H on 10-22-2014 at 14:46_

**Comment by apdavison** [comment]
see comments from @mattions on duplicate issue #201

**Comment by ashander** [comment]
Suggestion in #226 fixes version installated via pip -- fixed this in #227

**Comment by maxalbert** [comment]
Closing this issue because the fix in #227 prevents the installation of a django version which raises the `AppRegistryNotReady` exception. We can open a separate issue for Sumatra compatibility with recent versions of Django if required.
